packages-ts/gauntlet-terra-contracts/src/commands/contracts/multisig/updateAdmin.ts

```diff
@@ -0,0 +1,45 @@
+import { CATEGORIES } from '../../../lib/constants'
+import { isValidAddress } from '../../../lib/utils'
+import { AbstractInstruction, instructionToCommand } from '../../abstract/executionWrapper'
+
+type CommandInput = {
+  admin: string
+}
+
+type ContractInput = {
+  admin: string
+}
+
+const makeCommandInput = async (flags: any, args: any[]): Promise<CommandInput> => {
+  return {
+    admin: flags.admin,
+  } as CommandInput
+}
+
+const validateInput = (input: CommandInput): boolean => {
+  if (!isValidAddress(input.admin)) {
+    throw new Error('Admin address is not valid!')
+  }
+
+  return true
+}
+
+const makeContractInput = async (input: CommandInput): Promise<ContractInput> => {
+  return {
+    admin: input.admin,
+  } as ContractInput
+}
+
+// yarn gauntlet cw4_group:update_admin --admin=<NEW_ADMIN_ADDRESS> <CONTRACT_ADDRESS>
```

Comment:
@RodrigoAD where is the place to add these examples (in code)?

@sdrug why don't you also add some docs on these commands in `./docs`. It's empty currently but this is a good opportunity to start adding to it. Check out this rudimentary example from Solana: https://github.com/smartcontractkit/chainlink-solana/tree/develop/docs/multisig

```suggestion
```